commcare_connect/opportunity/forms.py

```diff
@@ -73,7 +74,10 @@ def clean_users(self):
         return split_users
 
 
-class OpportunityChangeForm(forms.ModelForm, OpportunityUserInviteForm):
+class OpportunityChangeForm(
```

Comment:
This adjustment is made because the `Method Resolution Order (MRO)` in Python prevented us from passing `org_slug` to `OpportunityUserInviteForm`. When attempting to pass `org_slug` to the superclass using `super()`, it raised an error since ModelForm does not accept `org_slug` in its keyword arguments.